addon/lib/shooter.js

```diff
@@ -198,10 +198,15 @@ const ShotContext = Class({
       }).bind(this)));
     }, this));
     this.interactiveWorker.port.on("textSelection", watchFunction(function (textSelection) {
-      this.activeClipName = this.shot.addClip({
-        createdDate: Date.now(),
-        text: textSelection
-      });
+      if (this.activeClipName){
+        let c = this.shot.getClip(this.activeClipName);
+        c.text = textSelection;
```

Comment:
Yes, it does happen currently. If you are viewing a text selection on the panel and you make a new text selection in the page, it happens.